### PR TITLE
<TextArea/> - InputArea should not add hover styles when focused and hovered

### PR DESCRIPTION
**Pull request by Faradey27**
### What changed

- InputArea should not add hover styles when focused and hovered

### Why it changed

- issue was created

---

- [x] Change is tested
- [x] Change is documented
- [x] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by Faradey27** [comment]
@shlomitc

**Comment by Faradey27** [comment]
@hepiyellow done

File: src/InputArea/InputArea.e2e.js [hunk 1]
```diff
@@ -17,9 +17,23 @@ describe('input area page', () => {
...
-    expect(inputAreaTestkit.isFocused()).toBe(false);
+    expect(inputAreaTestkit.isFocused()).toBeFalsy();
     await browser.actions().sendKeys(protractor.Key.TAB).perform();
-    expect(inputAreaTestkit.isFocused()).toBe(true);
+    expect(inputAreaTestkit.isFocused()).toBeTruthy();
   });
 
+  eyes.it('should show hover styles', async () => {
+    expect(await inputAreaTestkit.isHovered()).toBeFalsy();
+    await browser.actions().mouseMove(inputAreaTestkit.textArea()).perform();
```
Comment (hepiyellow) [inline src/InputArea/InputArea.e2e.js]: Please use `mouseEnter`
`import {  mouseEnter } from 'wix-ui-test-utils/protractor';`

File: src/InputArea/InputArea.e2e.js [hunk 2]
```diff
@@ -17,9 +17,23 @@ describe('input area page', () => {
...
+    expect(await inputAreaTestkit.isHovered()).toBeTruthy();
+  });
+
+  eyes.it('should show hover and focus styles', async () => {
+    expect(await inputAreaTestkit.isHovered()).toBeFalsy();
+    expect(inputAreaTestkit.isFocused()).toBeFalsy();
+    await browser.actions().sendKeys(protractor.Key.TAB).perform();
```
Comment (hepiyellow) [inline src/InputArea/InputArea.e2e.js]: You can simply click on the textArea element, which will make it focused an hovered.
might add a `clickTextArea()` method to the driver, and remove the `textArea()` element getter.

Comment (hepiyellow) [reply src/InputArea/InputArea.e2e.js]: Only if the need focus+hover. And I think they don't.

To take a snapshot of focus-only you need TAB
snapshot of hover only - you need mouseEnter

File: src/InputArea/InputArea.e2e.js (hunk 1 above)
Comment (hepiyellow) [reply src/InputArea/InputArea.e2e.js]: Can you add documentation for it?
There is also mouseLeave.

**Pull request merged**